loris/resolver.py

```diff
@@ -381,9 +372,13 @@ def copy_to_cache(self, ident):
 
         logger.info("Copied %s to %s" % (source_url, local_fp))
 
-    def resolve(self, ident):
+    def delete_from_cache(self, ident):
```

Comment:
> Does it make sense for the SimpleFSResolver to purge items?

I found myself asking that as well. [Maybe](https://docs.google.com/document/d/1aRUwT0KNYwg9BcUhGsnPomaMNZCXcaZYutfXHTXO4_o/edit)? I can see a use case for sending `DELETE` to `/identifier`, (and then why not `PUT` or `POST` as well?). As the doc I linked to suggests, I've always kind of wanted to do this.

I think moving the source image cache into a separate dict-like class is probably a good first step, and then we could revisit where we want to expose it. I suspect your instinct to do it in the resolver might be right, but that's layering on a lot of new behavior, and we might want to rename the Resolver class to something like `ImageRepository`.